R/lazy-select-query.R

```diff
@@ -166,8 +173,24 @@ sql_build.lazy_select_query <- function(op, con, ..., sql_options = NULL) {
 
   alias <- remote_name(op$x, null_if_local = FALSE) %||% unique_subquery_name()
   from <- sql_build(op$x, con, sql_options = sql_options)
+
+  selects <- op$select
+
+  if (!is.logical(op$distinct)) {
```

Comment:
I think it would be more clear to say what `op$distinct` is, rather than what it's not.